src/containers/guest-home-page/signup-dialog/SignupDialog.jsx

```diff
@@ -85,7 +86,14 @@ const SignupDialog = ({ type }) => {
   useEffect(() => {
     setNeedConfirmation(isDirty)
   }, [isDirty, setNeedConfirmation])
-
+  const { isSmallScreen, isMediumScreen, isLargeScreen, isXLargeScreen } =
+    useBreakpointsGoogle()
+  const getButtonWidth = () => {
+    if (isSmallScreen) return 330
+    if (isMediumScreen) return 333
+    if (isLargeScreen) return 340
+    if (isXLargeScreen) return 394
+  }
```

Comment:
We can use `useCallback` or object style for definition of size like
```suggestion
  const getButtonWidth =useMemo(()=> ({
    isSmallScreen: 330,
    isMediumScreen: 333,
    isLargeScreen: 340,
    isXLargeScreen: 394
  }), [] )
```